Source/WebKit/UIProcess/API/Cocoa/WKWebpagePreferences.mm

```diff
@@ -828,4 +840,36 @@ - (BOOL)_allowsJSHandleCreationInPageWorld
     return _websitePolicies->allowsJSHandleCreationInPageWorld();
 }
 
+- (void)setSecurityRestrictionMode:(WKSecurityRestrictionMode)mode
+{
+    if (!isEnhancedSecurityFeatureEnabled())
+        return;
```

Comment:
I am curious why we are returning early here. Shouldn't we set the mode unconditionally?